src/main/resources/reference.conf

```diff
@@ -156,4 +156,46 @@ cassandra-snapshot-store {
       parallelism-max = 8
     }
   }
+}
+
+cassandra-query-journal {
+  # Implementation class of the Cassandra ReadJournalProvider
+  class = "akka.persistence.cassandra.query.journal.CassandraReadJournalProvider"
+
+  # Stream queries database in defined `refresh-interval`.
+  refresh-interval = 3s
+
+  # How many events to fetch in one query (replay) and keep buffered until they
+  # are delivered downstreams.
+  max-buffer-size = 100
+
+  # Target partition size of index tables. Must match cassandra-journal configuration.
+  target-partition-size = 500000
```

Comment:
I don't have a strong opionion about the how but it's unfortunate if same things must be entered twice, when it's really a write side property.

Common section is not that easy since there can be many write plugins and many query plugins.